src/docs/components/docs-topbar/docs-topbar.css

```diff
@@ -1,19 +1,34 @@
-.docs-topbar::part(container) {
-  margin: 0 var(--ld-sp-4);
+.docs-topbar {
   min-width: 20rem;
-  padding: 0 var(--ld-sp-24);
+
+  &::part(container) {
+    margin: 0 var(--ld-sp-4);
+    padding: 0 var(--ld-sp-24);
+  }
 }
 
 .docs-topbar__logo {
   display: block;
+  margin-left: calc(-1 * var(--ld-sp-6));
```

Comment:
In the breakpoint where the navigation is permanently visible, the logo does now no longer align with the navigation (icons). Same goes for the icons/links on the right side - maybe we should add another breakpoint with increased `.docs-topbar::part(container)` padding for the in-between breakpoint?